src/main/java/net/dries007/tfc/config/ServerConfig.java

```diff
@@ -365,6 +367,10 @@
         enablePlacingItems = builder.apply("enablePlacingItems").comment("If true, players can place items on the ground with V.").define("enablePlacingItems", true);
         usePlacedItemWhitelist = builder.apply("usePlacedItemWhitelist").comment("If true, the tag 'tfc:placed_item_whitelist' will be checked to allow items to be in placed items and will exclude everything else.").define("usePlacedItemWhitelist", false);
 
+        innerBuilder.pop().push("dispenser");
+
+        dispenserEnableLighting = builder.apply("dispenserEnableAutomation").comment("If true, dispensers will be able to light blocks.").define("dispenserEnableAutomation", true);
```

Comment:
config value `dispenserEnableLighting` should match the name `dispenserEnableAutomation`